<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>50</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>950</width>
    <height>50</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>950</width>
    <height>50</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#btn_cartAmountDec {
	border-image: url(:/2-2_cartItem/src/cartMinus_O.png);
}

#btn_cartAmountDec:hover {
	border-image: url(:/2-2_cartItem/src/cartMinus_H.png);
}

#btn_cartAmountDec:disabled {
	border-image: url(:/2-2_cartItem/src/cartMinus_D.png);
}

#btn_cartAmountInc {
	border-image: url(:/2-2_cartItem/src/cartAdd_O.png);
}

#btn_cartAmountInc:hover {
	border-image: url(:/2-2_cartItem/src/cartAdd_H.png);
}

#btn_cartRemove {
	border-image: url(:/2-2_cartItem/src/cartRemove_O.png);
}

#btn_cartRemove:hover {
	border-image: url(:/2-2_cartItem/src/cartRemove_H.png);
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QPushButton" name="btn_cartRemove">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="menuName_">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>menuName_</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="optionList_">
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>optionList_</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_cartAmountDec">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="quantity">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>quantity</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_cartAmountInc">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="itemPrice">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>itemPrice</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../UI/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btn_cartAmountInc</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>cartItemAmount_Increase()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>832</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>830</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_cartAmountDec</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>cartItemAmount_Decrease()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>720</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>720</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_cartRemove</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>cartItem_Remove()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>21</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>-51</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>cartItem_Remove()</slot>
  <slot>cartItemAmount_Increase()</slot>
  <slot>cartItemAmount_Decrease()</slot>
 </slots>
</ui>
